src/wizard.scss

```diff
@@ -9,6 +9,20 @@ $fd-wizard-step-no-label-max-size: 4rem !default;
 $fd-wizard-step-indicator-stacked-offset: -1 * ($fd-wizard-step-indicator-size - $fd-wizard-step-indicator-spacing + 0.25rem) !default;
 $fd-wizard-stacked-upcoming-step-offset: 2.375rem !default;
 
+@mixin fd-wizard-completed-step($block) {
```

Comment:
do you need to pass $block as argument?

Reply:
You can also set the default value and then you don't have to pass it when you include the mixin.

Reply:
@InnaAtanasova yes, $block was not needed. Removed.